.dev/tests/cypress/integration/customizer/customizer.spec.js

```diff
@@ -0,0 +1,163 @@
+
+	const socialAccounts = {
+		facebook: 'https://www.facebook.com',
+		twitter: 'https://www.twitter.com',
+		instagram: 'https://www.instagram.com',
+		linkedin: 'https://www.linkedin.com',
+		xing: 'https://www.xing.com',
+		pinterest: 'https://www.pinterest.com',
+		youtube: 'https://www.youtube.com',
+		spotify: 'https://www.spotify.com',
+		github: 'https://www.github.com',
+	};
+
+	before( () => {
+		loginToSite();
+	} );
+
+	beforeEach( () => {
+		cy.visit( Cypress.env( 'localTestURL' ) + '/wp-admin/customize.php' );
+		cy.get( 'body' ).should( 'have.class', 'wp-customizer' );
+		cy.frameLoaded( '[name="customize-preview-0"]' );
+	} );
+
+	it( 'Test Site title, description and a custom logo', () => {
+		cy.get( '#accordion-section-title_tagline' ).click();
+
+		// Custom Logo
+		cy.get( '#customize-control-custom_logo .attachment-media-view .upload-button' ).click();
+		upload.newLogo();
+		cy.get( '.media-frame-toolbar .media-toolbar-primary button.button-primary' ).should( 'not.have.class', 'disabled' ).click();
+		cy.get( '.media-toolbar button.button-secondary' ).contains( 'Skip cropping' ).click();
+
+		// Site Title
+		cy.get( '#_customize-input-blogname' ).clear().type( atts.title );
+
+		// Site Tagline
+		cy.get( '#_customize-input-blogdescription' ).clear().type( atts.tagline );
+
+		saveCustomizerSettings();
+
+		cy.reload();
+
+		cy.frameLoaded( '[name="customize-preview-0"]' );
+		cy.iframe( 'iframe[name="customize-preview-0"]' ).find( '.custom-logo-link img' ).should( 'have.attr', 'src' ).and( 'match', /150x150/ );
+		cy.iframe( 'iframe[name="customize-preview-0"]' ).find( 'h1.site-title' ).contains( atts.title );
+		cy.iframe( 'iframe[name="customize-preview-0"]' ).find( 'span.site-description' ).contains( atts.tagline );
+
+		// Test Hidden title and tagline
+		cy.get( '#accordion-section-title_tagline' ).click();
+		cy.get( '#_customize-input-hide_site_title_checkbox' ).click();
+		cy.get( '#_customize-input-hide_site_tagline_checkbox' ).click();
+		saveCustomizerSettings();
+
+		cy.reload();
+
+		cy.iframe( 'iframe[name="customize-preview-0"]' ).find( 'h1#site-title' ).should( 'not.exist' );
+		cy.iframe( 'iframe[name="customize-preview-0"]' ).find( 'span.site-description' ).should( 'not.exist' );
+	} );
+
+	it( 'Should switch design styles as intended', () => {
+		// Start at the bottom of the list
+		designStyles.reverse().forEach( designStyle => {
+			cy.get( '#accordion-section-colors' ).click();
+			cy.get( 'label[for="_customize-input-design_style_control-radio-' + designStyle.toLowerCase() + '"]' ).click( { force: true } );
+			saveCustomizerSettings();
+			cy.reload();
+			cy.frameLoaded( '[name="customize-preview-0"]' );
+			cy.iframe( 'iframe[name="customize-preview-0"]' ).should( 'have.class', 'is-style-' + designStyle.toLowerCase() );
+		 } );
+
+		// Loop over design style labels
+		// cy.get( '#customize-control-design_style_control .customize-inside-control-row' ).then( $designStyle => {
```

Comment:
If you don't intend on implementing these, I would simply remove the commented out code.